tests/gtest/avifallocationtest.cc

```diff
@@ -188,13 +188,25 @@ TEST(EncodingTest, MaximumInvalidDimensions) {
                AVIF_RESULT_UNSUPPORTED_DEPTH);
 }
 
+#if defined(__clang__) && defined(__has_feature)
```

Comment:
Optional: The `defined(__clang__)` test probably can be omitted. The `defined(__has_feature)` test should be sufficient.

Reply:
Probably. Done